wolfcrypt/src/aes.c

```diff
@@ -9319,7 +9101,7 @@ static void roll_auth(Aes* aes, const byte* in, word32 inSz, byte* out)
         out[0] ^= ((inSz & 0xFF00) >> 8);
         out[1] ^=  (inSz & 0x00FF);
     }
-    else if (inSz <= 0xFFFFFFFF) {
+    else if (inSz <= 0xFEFFFFFF) {
```

Comment:
@toddouska : I think I should remove this change. We discussed, but I did not finalize my investigation.